path_finding.asciidoc

```diff
@@ -3,6 +3,22 @@
 
 Payment delivery on the Lightning Network depends on finding a path from the sender to the recipient, a process called _path finding_.
 
+=== Path finding in the Lightning protocol suite
```

Comment:
I am still not sure if we might want to switch to something like `Finding Liquidity` or `liquidity finding` by the end of the day this issue is way more a transportation problem (we want to transport and amount `amt` from `S` to `R`